ads/logly.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {loadScript, validateData} from '../3p/3p';
+
+/**
+ * @param {!Window} global
+ * @param {!Object} data
+ */
+export function logly(global, data) {
+  validateData(data, ['adspotid']);
+
+  const d = global.document.createElement('div');
+  d.id = 'logly-lift-' + data['adspotid'];
+  global.document.getElementById('c').appendChild(d);
+
+  const url =
+    'https://l.logly.co.jp/lift_widget.js' +
+    `?adspot_id=${encodeURIComponent(data['adspotid'])}`;
+
+  loadScript(global, url, () => {
+    window.context.renderStart();
```

Comment:
you will need to call renderStart in your lift_widget.js when the ad is ready to be displayed

Reply:
fixed. I implemented `renderStart` and `noContentAvailable` in lift_widget.js 😃